docs/en/stack/ml/df-analytics/ml-dfa-regression.asciidoc

```diff
@@ -7,264 +7,167 @@
+== 6. Evaluate the result
+
+include::ml-dfa-shared.asciidoc[tag=dfa-evaluation-intro]
 
 You can measure how well the model has performed on your training data set by 
 using the `regression` evaluation type of the 
-{ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API]. The mean squared 
-error (MSE) value that the evaluation provides you on the training data set is 
-the _training error_. Training the {regression} model means finding the 
+{ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API]. The 
+<<ml-dfanalytics-mse,mean squared error (MSE)>> value that the evaluation 
+provides you on the training data set is the _training error_. Training the 
+{regression} model – which is an iterative process – means finding the 
 combination of model parameters that produces the lowest possible training 
 error.
 
 Another crucial measurement is how well your model performs on unseen 
-data points. To assess how well the trained model will perform on data it has 
-never seen before, you must set aside a proportion of the training data set for 
-testing. This split of the data set is the testing data set. Once the model has 
-been trained, you can let the model 
+data. To assess how well the trained model will perform on data it has 
+never seen before, you must set aside a proportion of the training set for 
+testing (testing data set). Once the model is trained, you can let the it 
 predict the value of the data points it has never seen before and compare the 
 prediction to the actual value. This test provides an estimate of a quantity 
 known as the _model generalization error_.
 
-Two concepts describe how well the {regression} algorithm was able to learn the 
-relationship between the {feature-vars} and the {depvar}. _Underfitting_ is when 
-the model cannot capture the complexity of the data set. _Overfitting_ is when 
-the model is too specific to the training data set and is capturing details 
-which do not generalize to new data. A model that overfits the data has a 
-low MSE value on the training data set and a high MSE value on the testing 
-data set. For more information about the evaluation metrics, see 
-<<ml-dfanalytics-regression-evaluation>>.
-
-
-[discrete]
-[[dfa-regression-readings]]
-=== Further readings
-
-* https://github.com/elastic/examples/tree/master/Machine%20Learning/Feature%20Importance[Feature importance for {dfanalytics} (Jupyter notebook)]
-
-* https://github.com/elastic/examples/tree/master/Machine%20Learning/Regression%20Loss%20Functions[Regression loss functions (Jupyter notebook)]
-
-
-[discrete]
-[[ml-dfanalytics-regression-evaluation]]
-=== {regression-cap} evaluation
-
-include::ml-dfa-shared.asciidoc[tag=dfa-evaluation-intro]
-
-This evaluation type is suitable for evaluating {regression} models. The 
-{regression} evaluation type offers the following metrics to evaluate the model 
-performance:
+The {regression} evaluation type offers the following metrics to evaluate the 
```

Comment:
> The {regression} evaluation type offers the following metrics to evaluate the ...

I'm not sure there's enough detail in the separate "Loss functions for regression..." page to justify a separate page.  I suggest either (A) merging that content into this page, or (B) augmenting the content in that separate page (and covering all the evaluation types not just the loss ones). It seems to me that you need to have advanced/specific data science skills to know which evaluation types to use.

Reply:
@lcawl I think the issue here is that the loss functions in the separate page are not just for evaluating model performance but they have the power to influence model training by setting their parameters (for example delta in the case of Huber). This is why I thought it would be nice to mention them here in the model performance section and then describe them on a separate page in more detail where we explain when to use them and for what. The content on this page just explains their roles in the evaluation process, the description on the separate page goes beyond it and describes them as functions, not just passive values. For this reason, I still think there is value in keeping the regression loss functions in a separate page under Advanced concepts.

Reply:
So from re-reading that page, it seems like loss functions are used during model-building by all types of analytics and this can't be changed? Therefore likely not too necessary for anyone except the most curious advanced users to know about (if at all).
For regression analysis in particular, you can set it when you create the job, but it's recommended to use default values unless you're an advanced user.  
I can see why you're wanting to put this in a separate page, I'm just not sure there's enough info to justify it at this point (especially since it only really seems to be of interest to the regression use case if I'm reading that right). We'll definitely want to get a feel for when something gets split off into a separate page in this new model, but we can keep tweaking this as we figure that out with more content.

Reply:
@lcawl Thanks for your comment!

> I can see why you're wanting to put this in a separate page, I'm just not sure there's enough info to justify it at this point

I thought loss functions could be a separate page as we already have a stand-alone page under Advanced concepts with similarly short content `Hyperparameter optimization`. I used it as precedence. I don't think the concept of loss functions is easier to grasp than hyperparameter optimization. And the tuning of these functions is similarly only for users who are deeply familiar with them. This is why I think it is better to have them separately on another page, so they won't confuse the rest of the users. However, I see the point that you've made above. Do you think it would be better to add the function descriptions back?

Reply:
I think just go with what you've got for now and we can keep pondering and revising as we get more comfortable with our divisions of content!